sources/packages/web/src/views/student/StudentProfile.vue

```diff
@@ -2,13 +2,31 @@
   <student-page-container>
     <template #header>
       <header-navigator v-if="editMode" title="Student" subTitle="Profile" />
+      <row v-else align="center">
+        <v-col>
+          <p class="category-header-x-large">Create Your Profile</p>
+          <p>
+            Use your most up-to-date personal information.<br />
+            We'll use the same information to help you apply for financial aid.
```

Comment:
line 9 and 10 inside 2 <p> will avoid <br/>

Reply:
There will be a spacing difference. The `br` here is to keep like Figma.

![image](https://user-images.githubusercontent.com/61259237/182942079-d32cbc29-ddc3-4963-991d-e43a07eab161.png)

![image](https://user-images.githubusercontent.com/61259237/182942118-aaf68b32-6925-4a10-b11e-89bcaeae4b79.png)